test/must-equal-strict-spec.js

```diff
@@ -0,0 +1,482 @@
+var testHelper = require('../etc/test-helper.js');
```

Comment:
As noted in the pull request description, this file borrows from `must-equal-spec.js`.